{"name":"runTimeExceptionFailed","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nMethod ExpectedException.runTimeExceptionFailed()[pri:0, instance:com.dachan.testng.ExpectedException@681aad3b] should have thrown an exception of type class java.lang.RuntimeException","trace":"org.testng.TestException: \nMethod ExpectedException.runTimeExceptionFailed()[pri:0, instance:com.dachan.testng.ExpectedException@681aad3b] should have thrown an exception of type class java.lang.RuntimeException\r\n\tat org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)\r\n\tat org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:695)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1612856447730,"stop":1612856447731,"uuid":"0869f261-5fab-4f4e-b16d-275aadd836f5","historyId":"5aecc472b6884d0990cd9edcaf719f47","fullName":"com.dachan.testng.ExpectedException.runTimeExceptionFailed","labels":[{"name":"package","value":"com.dachan.testng.ExpectedException"},{"name":"testClass","value":"com.dachan.testng.ExpectedException"},{"name":"testMethod","value":"runTimeExceptionFailed"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AutoTest"},{"name":"subSuite","value":"com.dachan.testng.ExpectedException"},{"name":"host","value":"DESKTOP-T0RPK91"},{"name":"thread","value":"10912@DESKTOP-T0RPK91.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}